docs/notifications

```diff
@@ -0,0 +1,61 @@
+# Notifications
+
+*This page is about notifications displayed within the app. If you are interested in push notifications, you may want to read [this](https://docs.karrot.world/mobile.html) instead.*
+
+We distinguish between banners and toasts.
+
+## Banners
+Banners are notifications that are potentially shown for a long period of time. For instance, a banner could remind the user to sign an agreement which does not have a deadline. 
+
+Banners are shown at the top of the screen, right below the navigation bar. 
+
+To create a new banner, first add a new method to the `BannerUI` Vue component:
+
+```
```

Comment:
Do you get proper syntax highlighting with this? Otherwise, you might need to add a `js` language tag, similar to this file: https://raw.githubusercontent.com/yunity/karrot-frontend/master/docs/forms.md